tests/test_configurableActions.py

```diff
@@ -0,0 +1,226 @@
+        self.assertEqual(config.singleAction(), 3)
+
+    def testValidate(self):
+        configClass = self._createConfig(default={"test1": TestAction1, "test2": TestAction2,
+                                                  "test3": TestAction3}, singleDefault=TestAction1)
+        config = configClass()
+        config.validate()
+
+    def testFreeze(self):
+        configClass = self._createConfig(default={"test1": TestAction1, "test2": TestAction2},
+                                         singleDefault=TestAction1)
+        config = configClass()
+        config.freeze()
+
+        with self.assertRaises(FieldValidationError):
+            config.actions.test3 = TestAction3
+
+        with self.assertRaises(FieldValidationError):
+            config.actions.test1.var = 2
+
+        with self.assertRaises(FieldValidationError):
+            config.actions.test2.var = 0
+
+        with self.assertRaises(FieldValidationError):
+            config.singleAction = TestAction2
+
+        with self.assertRaises(FieldValidationError):
+            config.singleAction.var = 3
+
+    def testCompare(self):
+        configClass = self._createConfig(default={"test1": TestAction1, "test2": TestAction2},
+                                         singleDefault=TestAction1)
+        config = configClass()
+        config2 = configClass()
+
+        self.assertTrue(config.compare(config2))
+
+        # Test equality fail for ConfigurableActionsStructField
+        config3 = configClass()
+        config3.actions.test1.var = 99
+        self.assertFalse(config.compare(config3))
+
+        # Test equality fail for ConfigurableActionsField
+        config4 = configClass()
+        config4.singleAction.var = 99
+        self.assertFalse(config.compare(config4))
+
+    def testSave(self):
+        # This method will also test rename, as it is part of the
+        # implementation in pex_config
+        ioObject = StringIO()
+        configClass = self._createConfig(default={"test1": TestAction1},
+                                         singleDefault=TestAction1)
+        config = configClass()
+
+        config.saveToStream(ioObject)
+        loadedConfig = configClass()
+        loadedConfig.loadFromStream(ioObject.read())
+        self.assertTrue(config.compare(loadedConfig))
+        # Be sure that the fields are actually there
+        self.assertEqual(loadedConfig.actions.test1.var, 0)
+        self.assertEqual(loadedConfig.singleAction.var, 0)
+
+    def testToDict(self):
+        """Test the toDict interface"""
+        configClass = self._createConfig(default={"test1": TestAction1},
+                                         singleDefault=TestAction1)
+        config = configClass()
+        self.assertEqual(config.toDict(), {'actions': {'test1': {'var': 0}}, 'singleAction': {'var': 0}})
+
+
+def setup_module(module):
+    lsst.utils.tests.init()
+
+
+class MatchMemoryTestCase(lsst.utils.tests.MemoryTestCase):
+    pass
```

Comment:
Name of this class looks like a copy-paste relic.  I'm also not sure if it's useful, or just old boilerplate.